docs/sources/mimir/_index.md

```diff
@@ -12,18 +15,49 @@ keywords:
   - observability
 menuTitle: Grafana Mimir
 title: Grafana Mimir documentation
-weight: 1
+hero:
+  title: Grafana Mimir
+  level: 1
+  image: /media/docs/mimir/GrafanaLogo_Mimir_icon.png
+  width: 100
+  height: 100
+  description: Grafana Mimir is an open source software project that provides horizontally scalable, highly available, multi-tenant, long-term storage for Prometheus.
+cards:
+  title_class: pt-0 lh-1s
+  items:
+    - title: Get started
+      href: ./get-started/
+      description: Get started quickly with Grafana Mimir's extensive documentation, tutorials, and deployment tooling. Use the monolithic mode to get up and running with just one binary and no added dependencies.
+      height: 24
+    - title: Set up and configure
+      href: ./configure/about-configurations/
+      description: Set up Grafana Mimir with Helm, Puppet, or Jsonnet and Tanka. Migrate to Grafana Mimir from Cortex, Thanos, or Prometheus. Configure Grafana Mimir through the use of a YAML-based configuration file or CLI flags and learn about the common configurations.
```

Comment:
`./configure/about-configurations/` isn't about Helm, Puppet etc, it's just a page explaining the YAML configuration

Reply:
It seems like we're currently grouping the "Set up" and "Configure" docs together here and perhaps conflating the content of these two sections. I'm going to split them into two separate tiles.